extContractsGroup.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "contivModel",
+	"objects": [
+		{
+			"name": "extContractsGroup",
+			"type": "object",
+			"key": [ "contractsGroupName" ],
```

Comment:
this would mean contract group name is global... 
What are the options? It can be qualified by tenant or be left as is.